charter.md

```diff
@@ -53,15 +53,227 @@ The following responsibilities belong to the Steering Committe, but may be deleg
+
+At least two (2) and not more than three (3) members shall be selected for
+this duty.
+
+* One shall be the longest-serving outgoing committee member. If multiple, the
+  incoming committee may select between them.
+* The second shall be selected by the incoming committee at their discretion.
+* The incoming committee, at their discretion, may select a third.
+
+#### Dissolution by vote
+
+Dissolution requires an affirmative vote of a **_two-thirds supermajority of
+the [fixed membership of the committee](#composition)_**, and must be unanimous
+when the committee has four (4) or fewer members.
+
+Example:
+
+* 7 (members) / 3 = 2.333...
+* 2.333... * 2 = 4.666...
+* Round up to the nearest whole number (5)
+* An affirmative vote of 5 members would be required to dissolve the committee
+
+The committee must specify a date of dissolution.
+If the committee is dissolved, all seats are vacated on the date specified.
+
+#### Dissolution by loss of committee
+
+If at any time a **_majority of the
+[fixed membership of the committee](#composition)_**'s seats are vacant, the entire committee will be dissolved.
+
+Example:
+
+* 7 (seats) / 2 = 3.5
+* Round up to the nearest whole number (4)
+* If 4 committee seats are vacant, the entire committee will dissolve
+
+The number of vacant seats will be evaluated prior to any replacement of
+previously-seated members.
+
+### Removal
+
+#### No confidence
+
+A Steering Committee member may be removed by a three-fourths vote of no
+confidence from other Steering Committee members.
+
+The call for a vote of no confidence will happen in a public Steering Committee meeting and a
+GitHub issue in kubernetes/steering by a current member and must be seconded by
+another current member.
+
+Once a vote of no confidence has been called, the committee will notify the
+community through the following channels:
+
+* the [community mailing list][dev-list]
+* the [Steering Committee public mailing list][steering-public-list]
+
+This notification should include a link to the aforementioned GitHub issue.
+
+There will be a period of one week for members of the community to reach
+out to Steering Committee members to provide feedback, which can include
+non-binding votes.
+
+Community members may provide feedback by the following methods:
+
+* commenting on the GitHub issue
+* replying to the notification email
+* sending an email to the
+  [Steering Committee private mailing list][steering-private-list]
+* sending a message to individual committee members
+
+After this feedback period, Steering Committee members must vote on the issue
+within 48 hours.
+
+[dev-list]: mailto:dev@kubernetes.io
+[steering-private-list]: mailto:steering-private@kubernetes.io
+[steering-public-list]: mailto:steering@kubernetes.io
+
+## Voting
+
+In the course of the committee's operations, members will be expected to vote
+on decisions within the body's purview.
+
+These votes may proceed on agreed-upon platforms by the committee, such as:
```

Comment:
Is this saying the vote will/can take place in multiple venues? "platforms"

Feels like it should take place in 1 for clarity.

Reply:
@kikisdeliveryservice -- As an example, a vote may be initiated on a meeting, with the actual vote captured as an artifact of a PR or issue.

Would it be useful to say something around the vote being called for on any of the accepted platforms, but needing to be captured within an issue or PR (in the case of public votes)?

Reply:
That clarifies it for me 🙂